bibo/internals.py

```diff
@@ -192,3 +195,50 @@ def complete_path(ctx, args, incomplete):
         if os.access(path, os.R_OK):
             options.append((path, os.path.basename(path)))
     return options
+
+
+def match_case(substring, target):
+    """
+    Return the string `substring` with the same case as in the `target`
+    string.
+    """
+    index = target.lower().find(substring.lower())
+    if index < 0:
+        msg = "Failed to match case: {} not in {}".format(substring, target)
+        raise ValueError(msg)
+    return target[index : (index + len(substring))]
+
+
+_ansi_bold = "\033[1m"
+_ansi_unbold = "\033[22m"
```

Comment:
Eh, conventions. Maybe something capsy and at the top of the file? Or, is this allowed in python-styling conventions these days?

Reply:
_AGREE :-)
Will do.